x-pack/plugins/security/public/account_management/user_profile/user_profile.tsx

```diff
@@ -301,6 +312,32 @@ const UserSettingsEditor: FunctionComponent<UserSettingsEditorProps> = ({
     );
   };
 
+  const deprecatedWarning = idSelected === 'space_default' && (
+    <>
+      <EuiSpacer size="s" />
+      <EuiCallOut
+        title={i18n.translate(
+          'xpack.security.accountManagement.userProfile.deprecatedSpaceDefaultTitle',
+          {
+            defaultMessage: 'Space default settings will be deprecated in 10.0',
+          }
+        )}
+        color="warning"
+        iconType="warning"
+      >
+        <p>
+          {i18n.translate(
+            'xpack.security.accountManagement.userProfile.deprecatedSpaceDefaultDescr',
```

Comment:
__nit:__ Just for the consistency with other i18n labels, it's just few more symbols! 😄 
```suggestion
            'xpack.security.accountManagement.userProfile.deprecatedSpaceDefaultDescription',
```